src/components/apphost.js

```diff
@@ -310,9 +311,11 @@ define(["appSettings", "browser", "events", "htmlMediaHelper"], function (appSet
 
     var deviceId;
     var deviceName;
+    var appName = "Jellyfin Web";
+    var appVersion = "10.3.6";
```

Comment:
If this the only place where the version is kept in the Web UI?

Reply:
yes